tools/generate-component-template.js

```diff
@@ -9,6 +10,9 @@ const componentName = process.argv[2];
 // Grab and validate appName from script args
 const appName = process.argv[3];
 
+// Check if new templates flag is present
+const newTemplates = process.argv[4] == 'new';
```

Comment:
this would work, but might be simpler to just have `npm run generate-legion <same args as before>`